### PR TITLE
refactor: run rustfmt with `wrap_comments` enabled

### PR DESCRIPTION
**Pull request by akhi3030**
Only changes comments, no other changes.

Related: #8490

### CODE REVIEW

**Review by nikurt — APPROVED** [review]
Haven't read all the changes, but they seem to be wrapping the comments.

I like the consistency.

File: chain/chain/src/flat_storage_creator.rs [hunk 1]
```diff
@@ -203,11 +212,13 @@ impl FlatStorageShardCreator {
...
-                    // If it holds, deltas for all blocks after final head are saved to disk, because they have bigger
-                    // heights than one on which we launched a node. Check that it is true:
+                    // If it holds, deltas for all blocks after final head are saved to disk,
+                    // because they have bigger heights than one on which we
+                    // launched a node. Check that it is true:
                     for height in final_height + 1..=chain_store.head()?.height {
-                        // We skip heights for which there are no blocks, because certain heights can be skipped.
-                        // TODO (#8057): make `get_all_block_hashes_by_height` return empty hashmap instead of error
+                        // We skip heights for which there are no blocks, because certain heights
+                        // can be skipped. TODO (#8057): make
```
Comment (nagisa) [inline chain/chain/src/flat_storage_creator.rs:220]: This isn’t great. Not sure how to best handle it… maybe an empty comment line before the `TODO`?

File: chain/chain/src/tests/simple_chain.rs [hunk 2]
```diff
@@ -39,8 +39,9 @@ fn build_chain() {
     // ‘nightly’ feature enabled:
     //
     //     cargo install cargo-insta
-    //     cargo insta test --accept -p near-chain                    -- tests::simple_chain::build_chain
-    //     cargo insta test --accept -p near-chain --features nightly -- tests::simple_chain::build_chain
+    //     cargo insta test --accept -p near-chain                    --
```
Comment (nagisa) [inline chain/chain/src/tests/simple_chain.rs:42]: This would probably have been handled better if it was inside a markdown code block…? Not sure.

**Review by nagisa — COMMENTED** [review]
https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=wrap_co#comment_width should be set to the same width as the source code, rather than being left at the default 80 IMO.

Otherwise I haven’t looked through the PR too thoroughly but I did flag a couple of general issues which I think are prone to happen across the codebase.

**Comment by akhi3030** [comment]
I believe this PR is no longer actively being worked on.

**Pull request closed without merging**